config/202-gateway.yaml

```diff
@@ -34,6 +34,8 @@ spec:
       protocol: HTTPS
     hosts:
     - "*"
+    tls:
+      mode: PASSTHROUGH
```

Comment:
this seems to mean that SNI is used to route, is it always there?

Reply:
Docs for this option are scant: https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Server.TLSOptions.TLSmode

I'm also curious how this value was chosen. Is it intended to have the same behavior as before?